src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,79 +1,72 @@
 package techcourse.myblog.domain;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 import java.util.Objects;
 
+@Entity
 public class Article {
     private static final String DEFAULT_COVER_URL = "/images/pages/index/study.jpg";
     private static final int NO_COVER_URL = 0;
-    private static long NEXT_ID = 1L;
 
-    private long articleId;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
     private String title;
-    private String contents;
     private String coverUrl;
+    private String contents;
 
     public Article() {
     }
 
-    public Article(final long articleId, final String title, final String contents, final String coverUrl) {
-        this.articleId = articleId == 0 ? NEXT_ID++ : articleId;
-        this.title = title;
-        this.contents = contents;
-        this.coverUrl = (coverUrl.length() != NO_COVER_URL) ? coverUrl : DEFAULT_COVER_URL;
+    public long getId() {
+        return id;
     }
 
-    public boolean matchId(long articleId) {
-        return this.articleId == articleId;
+    public String getTitle() {
+        return title;
     }
 
-    public void update(Article updatedArticle) {
-        this.articleId = updatedArticle.getArticleId();
-        this.title = updatedArticle.getTitle();
-        this.contents = updatedArticle.getContents();
-        this.coverUrl = updatedArticle.getCoverUrl();
+    public String getCoverUrl() {
+        return coverUrl;
     }
 
-    public long getArticleId() {
-        return articleId;
+    public String getContents() {
+        return contents;
     }
 
-    public String getTitle() {
-        return title;
+    public void setId(long id) {
+        this.id = id;
     }
 
-    public String getContents() {
-        return contents;
+    public void setTitle(String title) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요? setter 메서드는 도메인의 핵심 개념이나 의도를 코드에서 사라지게 해요.

```suggestion
    public void changeTitle(String title) {
```